src/styles/_catalog.scss

```
@@ -1,6 +1,10 @@
+@import "utils/variables";
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: $card-width $card-width $card-width $card-width;
+  column-gap: $cards-distance;
```

You can add gap to vertical and horizontal spaces between elements instead of setting margin bottom on each card.
```suggestion
  gap: $cards-distance;
```

---

I changed `column-gap` to `gap`.